### PR TITLE
Next.js

### PR DESCRIPTION
**Pull request by rkatic**
Still to do:

- [x] Font awsame
- [x] Remove unused dependencies.

Do do in separate PR(s):
- Build global style in single css in production.
- Server side Fetching (maybe in separate PR)

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT=7000
```
Comment (mateoKaradza) [inline .env]: this file should be in .gitignore

Comment (rkatic) [reply .env]: It is, but not at same time, apparently. :(

File: package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "babel-eslint": "^7.1.0",
     "babel-jest": "^17.0.0",
     "babel-loader": "^6.2.7",
+    "babel-plugin-wrap-in-js": "^1.1.1",
```
Comment (kotarac) [inline package.json]: what's this for?
it doesn't have a git repository behind the npm package.
this is what I get when npm infoing it
```
{ name: 'babel-plugin-wrap-in-js',
  description: 'Babel plugin to wrap file in an exported js string',
  'dist-tags': { latest: '1.1.1' },
  versions: [ '1.1.0', '1.1.1' ],
  maintainers: [ 'davibe <dade@dadeb.it>' ],
  time: 
   { modified: '2017-01-31T09:07:38.697Z',
     created: '2017-01-31T08:34:26.475Z',
     '1.1.0': '2017-01-31T08:34:26.475Z',
     '1.1.1': '2017-01-31T09:07:38.697Z' },
  keywords: [ 'babel-plugin', 'js', 'wrap' ],
  repository: {},
  author: 'Davide Bertola <dade@dadeb.it>',
  license: 'MIT',
  readmeFilename: '',
  version: '1.1.1',
  main: 'index.js',
  scripts: {},
  dependencies: 
   { 'babel-plugin-transform-strict-mode': '^6.6.5',
     'babel-template': '^6.7.0' },
  devDependencies: { 'babel-core': '^6.7.2' },
  gitHead: 'd114fa9fa3f1a5c04634cf15efee2693dda924e4',
  dist: 
   { shasum: '437a86a92c0f18ef29ea92184c978848d0480b20',
     tarball: 'https://registry.npmjs.org/babel-plugin-wrap-in-js/-/babel-plugin-wrap-in-js-1.1.1.tgz' },
  directories: {} }
```

Comment (rkatic) [reply package.json]: It is used to make styles importable on both, server and client rendering.
The fact its package is not linked to the github repo is unfortunate (checked the code tho).
To be safe, we can fix its version.

**Pull request merged**